css/responsive/tablet.styl

```diff
@@ -41,11 +41,8 @@ for col in 1..$cols
 // ARCHIVE EDITORIAL
 
 #posts-archive-editorial
-  height: 100%
-  min-height: 600px
   .col
-    height: 100%
-    min-height: 600px
+    min-height: 620px
```

Comment:
there isnt anymore funky stuff in this feature branch. this is just forcing 620 height so smaller images still vertically center (I hope)